src/components/button/Button.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./Button.css";
+
+const Button = ({ name, className, onClick }) => {
```

Comment:
use {...rest} instead of {name, className, onClick} so you can pass all button keys as props not just these three